src/flow-spec/IFlow.ts

```diff
@@ -27,7 +27,7 @@ export interface IFlow {
   /**
    * A globally unique identifier for this Flow.  (See UUID Format: https://floip.gitbook.io/flow-specification/flows#uuid-format)
    *
-   * @TJS-pattern ^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$
+   * @pattern ^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$
```

Comment:
FYI @markboots , it seems like the `@TJS-` annotations are not working with `ts-json-schema-generator`, but just removing the prefix is working. I don't know why, and I don't see a doc explaining this behaviour so far. But after seeing this issue, I just tried https://github.com/vega/ts-json-schema-generator/issues/445

Reply:
cc: @bzabos & @poojahpatil90